setup/README.md

```diff
@@ -6,7 +6,7 @@ The purpose of this script is to sequentially deploy all the dependencies (i.e.
 
   - Kubernetes 1.15+ with Beta APIs
   - Helm 3.2.1+
-  - [Tier 2 Setup](https://github.com/pravega/pravega-operator#set-up-tier-2-storage)
+  - LongTerm Storage ([options for long term storage](doc/longtermstorage.md))
```

Comment:
Broken link. Is the fix: `doc/longtermstorage.md` -> `../doc/longtermstorage.md` ?

Reply:
fixed